packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -32,18 +32,20 @@
   "portalGuardian": "0xa0Ee7A142d267C1f36714E4a8F75612F20a79720",
   "finalizationPeriodSeconds": 2,
   "fundDevAccounts": true,
-  "l2GenesisBlockBaseFeePerGas": "0x1",
+  "l2GenesisBlockBaseFeePerGas": "0x3b9aca00",
```

Comment:
That's a very significant increase - it will take a fair while for that to drop back down to the minimum. It appears to be what's causing the test failures. :)

Reply:
Was intended to be able to test EIP1559 params in canyon. Dropping for op-e2e b/c op-e2e is unable to handle int.